msal-dotnet-articles/advanced/backup-authentication-system.md

```diff
@@ -1,26 +1,28 @@
 ---
 title: Backup authentication system
+description: "The Azure Active Directory backup authentication system enables the caching of credentials processed by the Evolved Security Token Service (ESTS) in order to provide resiliency during outages in Azure AD's authentication services."
 ---
 
 # Backup authentication system
 
-The Azure AD backup authentication system enables the caching of credentials process by ESTS in order to provide resiliency during outages in Azure AD's authentication services. In order to assist in speeding up the token retrieval from the backup authentication system, MSAL will provide a routing hint in the form of a header or an extra query parameter in authentication requests sent to ESTS. MSAL will attempt to do this for most of the authentication scenarios but there will come a situation where MSAL is not able to provide this hint due to the absence of user data. However, this can be resolved by the use of `WithCcsRoutingHint(string userObjectIdentifier, string tenantIdentifier)` or `WithCcsRoutingHint(string userName)`.
+The Azure Active Directory backup authentication system enables the caching of credentials processed by the Evolved Security Token Service (ESTS) in order to provide resiliency during outages in Azure AD's authentication services.
```

Comment:
If I remember, the docs folks didn't want to use ESTS term - only Azure AD (too technical probably?)

```suggestion
The Azure Active Directory has a backup authentication system that enables the caching of credentials in order to provide resiliency during outages in Azure AD's authentication services.
```